automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -37,14 +38,15 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   def clickLaunchAnalysis(): Unit = {
     openLaunchAnalysisModalButton.doClick()
     // defensive code to prevents test from failing. after click, expect to find either a Message or Analysis Modal
-    implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(3, Seconds)))
-    Try[String](eventually[String] {
-      find(CssSelectorQuery("body.broadinstitute-modal-open")).get.underlying.getTagName
-    }) match {
+    Try(
+      await condition (find(CssSelectorQuery(".broadinstitute-modal-open")).nonEmpty, 5)
```

Comment:
previous css string doesn't work. it failed to determine whether a Modal is open